jenkins/scripts/select-compiler.sh

```diff
@@ -8,13 +8,16 @@ if [ "$DONTSELECT_COMPILER" != "DONT" ]; then
   esac
 fi
 
+# get node version
+if [ -z ${NODE_MAJOR_VERSION+x} ]; then
+  NODE_VERSION="$(python tools/getnodeversion.py)"
+  NODE_MAJOR_VERSION="$(echo "$NODE_VERSION" | cut -d . -f 1)"
```

Comment:
During my work on updating the node-report CI job (https://github.com/nodejs/build/issues/1421#issuecomment-411469565) I had to add a parameter called `NODEJS_MAJOR_VERSION` to use the version selector script. This ends up as an environment variable of a same name (`NODEJS_MAJOR_VERSION`). 

https://github.com/nodejs/build/blob/d121160b7d38cecd98c794a2a9549b89708a4754/jenkins/scripts/VersionSelectorScript.groovy#L87-L88

I can make the job set `NODE_MAJOR_VERSION=NODEJS_MAJOR_VERSION`, but it would be one less line to maintain if I didn't have to (i.e. if the version selector and select compiler script used the same parameter/environment variable name).